### DISCUSSION TITLE
what to name the session for 2 subjects?

### DISCUSSION THREAD

**Discussion by shyt47** [TheLastBen/fast-stable-diffusion#499] [Q&A] [answered]
I an trying my first 2 subject session. I have myself and another person. I have 15 images of each with 2 different identifier names.  My question is what would I name my session in the colab? Also since its a man and a woman would I leave contains faces set to no? Also I name each subject its own unique name. Like manbbb( 1) and womnzzz (1) or should I name all the photos the same?

**Comment by TheLastBen** [comment, marked as answer]
each subject has to have a different identifier, as for the session name, it's unrelated to the training, it's just the name of the current workspace.
Train without using "contains_faces", set the text encoder to 60% and the total steps to 5000, set the images size to 640x640 in both the instance images cell and the training cell

**Reply by jysung** [reply]
Is there a reason for 640 x 640? Would it be different from 512 x 512, which I thought was supposedly efficient size for training.

**Reply by TheLastBen** [reply]
If you want more details, choose a higher resolution, the higher, the better, make sure the instance images are actually that size, not upscaled

**Reply by shyt47** [reply]
So what would the prompt look like? I tried xyz man with zyx woman, but I dont think its really working well. I may need to resume training a bit but some times it makes my man a woman.

**Reply by TheLastBen** [reply]
in the prompt add the term man before the the man and woman before the woman, and use (( ))
